solr/core/src/test/org/apache/solr/search/TestSolrCoreParser.java

```diff
@@ -146,10 +138,10 @@ public void testHandySpanQuery() throws IOException, ParserException {
       final Query clauseQuery = bq.clauses().get(ii).getQuery();
       switch (ii) {
         case 0:
-          assertTrue(unwrapSpanBoostQuery(clauseQuery) instanceof SpanOrQuery);
+          assertTrue(clauseQuery instanceof SpanOrQuery);
```

Comment:
personal preference
```suggestion
          assertThat(query, Matchers.instanceOf(SpanOrQuery));
```

Reply:
Not a super fan because it's fairly unrelated and the rest of the class uses this pattern, but I'll make the change in this method.

Reply:
we can leave it to be consistent.